examples/Benchmark.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
+@author: Michael Wu
+
+Benchmark test
+Giving performances of RF(scikit) and MultitaskDNN(Keras & TF)
+on datasets: muv, nci, pcba, tox21
+
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import sys
+import os
+import numpy as np
+import shutil
+import time
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+
+from deepchem.datasets import Dataset
+from deepchem import metrics
+from deepchem.metrics import Metric
+from deepchem.utils.evaluate import Evaluator
+from deepchem.models.keras_models.fcnet import MultiTaskDNN
+from deepchem.models.keras_models import KerasModel
+from deepchem.models.multitask import SingletaskToMultitask
+from deepchem.models.sklearn_models import SklearnModel
+from deepchem.models.tensorflow_models.fcnet import TensorflowMultiTaskClassifier
+from deepchem.models.tensorflow_models import TensorflowModel
+from deepchem.splits import RandomSplitter
+
+from muv.muv_datasets import load_muv
+from nci.nci_datasets import load_nci
+from pcba.pcba_datasets import load_pcba
+from tox21.tox21_datasets import load_tox21
+
+def benchmarkLoadingDatasets(base_dir_o, n_features = 1024, datasetName = 'all',
+                             model = 'all', reload = True, verbosity = 'high'):
+  
+  assert datasetName in ['all', 'muv', 'nci', 'pcba', 'tox21']
+  
+  if datasetName == 'all':
+    #currently not including the nci dataset
+    datasetName = ['muv','pcba','tox21']
+  else:
+    datasetName = [datasetName]
+      
+  if 'tox21' in datasetName:
+    print('-------------------------------------')
+    print('Benchmark test on dataset: tox21')
+    print('-------------------------------------')
+    base_dir = os.path.join(base_dir_o, "tox21")
+    time1 = time.time()
+    #loading datasets for tox21
+    tasks_tox21,datasets_tox21,transformers_tox21 = load_tox21(base_dir,
+                                                               reload=reload)
+    time2 = time.time()
+    #time2-time1 is the time(s) used for dataset loading
+    
+    #dataset splitting, built-in method in load_tox21
+    train_dataset, valid_dataset = datasets_tox21
+    #running model
+    tox21_train,tox21_valid = benchmarkTrainAndValid(base_dir,train_dataset,
+                                                valid_dataset,
+                                                tasks_tox21,transformers_tox21,
+                                                n_features, model, verbosity)
+    time3 = time.time()
+    #time3-time2 is the time(s) used for fitting and evaluating
+        
+    with open('/home/zqwu/deepchem/examples/results.csv','a') as f:
```

Comment:
Can you change this to make `out_file` an explicit argument? We should allow user to specify desired output location. If no `out_file` is specified, writing to `/tmp/results.csv` is a good compromise.